ctools.py

```diff
@@ -1,16 +1,25 @@
 from login import login
 from status import run_status
-from unlock import unlock, start_ssh
+from unlock import enable_telnet, start_ssh, disable_ssh
 from run_cmd import run_cmd
 from suspend_sync import suspend_filer_sync
 from unsuspend_sync import unsuspend_filer_sync
+from reset_password import reset_filer_password
+from gooey import Gooey, GooeyParser
 from cterasdk import *
 from argparse import ArgumentParser
 from getpass import getpass
 import logging, sys
 
-def set_logging(p_level=logging.INFO,log_file="log.txt"):
+""" If any args are present, run in CLI mode"""
+if len(sys.argv) >= 2:
```

Comment:
Understood but this check doesn't work when I move it into main as mentioned here:
https://github.com/chriskiehl/Gooey/issues/449#issuecomment-534056010

I could try this approach:
https://github.com/chriskiehl/Gooey/issues/296#issuecomment-500247320

Or I can just remove this check and use `--ignore-gooey` flag instead when I want to run it from CLI.
I found that an added benefit to this is that I can add `use_cmd_args=True` to the `@Gooey` decorator and then pass any arguments into the GUI when launched. I think this could be a better feature in the long run.